app/assets/javascripts/discourse/app/widgets/link.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable ember/no-private-routing-service */
```

Comment:
Could we use the public router service `urlFor()` here?

Reply:
Will try